draft-bbs-signatures.md

```diff
@@ -714,7 +714,7 @@ A cryptographic hash function that takes as an arbitrary octet string input and
 
 ## BLS12-381 Ciphersuite
 
-H
+HASH
 : SHAKE-256 as defined in [@!SHA3]
```

Comment:
Just a note we need to come back here and define how many bytes are drawn because this is an extendable output function